lib/lightning/jobs/trigger.ex

```diff
@@ -35,9 +35,8 @@ defmodule Lightning.Jobs.Trigger do
     belongs_to :job, Job
     belongs_to :upstream_job, Job
 
-    field :type, Ecto.Enum,
-      values: @trigger_types,
-      default: :webhook
+    field :type, Ecto.Enum, values: @trigger_types
+    # default: :webhook
```

Comment:
@taylordowns2000 @stuartc I did comment this out (didn't remove yet) because I want you to tell me what you think about it.